.github/workflows/coverage-check.yml

```diff
@@ -92,6 +93,7 @@ jobs:
                   "*scripts*" \
                   "src/dollar/mocks/*" \
                   "src/dollar/utils/*" \
+                  "src/old/*" \
```

Comment:
oldArtifacts, deprecatedArtifacts

Reply:
"Artifact" is basically a synonym of a contract or ABI representation (in terms of [truffle framework](https://trufflesuite.com/)) so it seems to be ambiguous

Anyway renamed `old` to `deprecated` https://github.com/ubiquity/ubiquity-dollar/commit/261eabed1099a8dbe4df56373be7068f254168d0